src/NuGet.Status/Controllers/AdminController.cs

```diff
@@ -118,9 +118,9 @@ private async Task<bool> InsertEntityToStorage(StorageService storageService, Ma
         {
             try
             {
-                var table = storageService.GetCloudTable();
-                var operation = TableOperation.Insert(entity);
-                await table.ExecuteAsync(operation);
+                TableClient tableClient = storageService.GetTableClient();
```

Comment:
Similar question here to my other one. Can we create and cache a single table client instead of recreating client for each insertion?